app/controllers/sessions_controller.rb

```diff
@@ -20,6 +20,14 @@ def create
     redirect_to session[:redirect_to] || admin_root_path
   end
 
+  def destroy
+    session[:current_user] = nil
+    session[:current_provider] = nil
+    session[:authenticated_until] = nil
+
+    redirect_to root_path
```

Comment:
If you click "back" in the browser window after signing out you can still view the last admin page you were on:

![log-out-still-logged-in](https://user-images.githubusercontent.com/282788/40842583-3e2dc240-65a6-11e8-984a-848a4d5b7349.gif)

This doesn't happen in Alaveteli:

![log-out-still-logged-in-wdtk](https://user-images.githubusercontent.com/282788/40842628-685210ee-65a6-11e8-94e0-371727647f07.gif)

If we can fix this easily then great, if not, I don't think its the end of the world to leave as-is?

Reply:
Think this would be because of turbolinks, will try: https://github.com/turbolinks/turbolinks#ensuring-specific-pages-trigger-a-full-reload

Reply:
Ah cool! If it doesn't work just make a new issue for this specific problem and get this merged :)

Reply:
Fixed in https://github.com/mysociety/foi-for-councils/pull/111/commits/5bc34a01e5b631dd4a43b23c2409ba2bbddf4721